packages/plugin-core/src/components/lookup/buttons.ts

```diff
@@ -80,6 +83,48 @@ function isSplitButton(button: DendronBtn) {
   return _.includes(["horizontal", "vertical"], button.type);
 }
 
+// TODO: make it into a utils method?
+const selectionToNoteProps = async (opts: {
```

Comment:
heuristic for making this into utils method is if this is used in at least 2 places